tests/integration/conftest.py

```diff
@@ -51,7 +51,11 @@
 
 def skip_if_linux(modeler: Modeler, test_name: str, element_not_available: str):
```

Comment:
I would rename this method to ``skip_if_core_service`` moving forward. What do you think?